Added solution for Scrambled-words challenge

gteti opened the pull request:
I've created this repo to add my proposed solution to the Reply training week1 Scrambled words challenge.

gteti committed:
Create README.md

Added folder README.md

gteti committed:
Add files via upload

Added original challenge files and proposed solution created in Jupyter/Python.

gteti commented:
Anyone please check if it is good to merge, Thanks

gmacario commented:
Sorry but I cannot merge this PR unless it passes some automated tests such as <https://github.com/github/super-linter>

gteti commented:
#2 has been merged.

gteti committed:
Create README.md

Creation README for Dig into the Blood of reply training cybersecurity challenge 2021

gteti committed:
Add files via upload

Added original files and solution for this challenge.

gteti committed:
Delete training/week1/miscellaneous/dig_into_the_blood directory

Already present a PR about this challenge.

gmacario commented on training/week1/coding/scrambled-words/README.md:
```diff
@@ -0,0 +1,19 @@
+
+Use the _.ipynb_ file to resolve this enigma. I converted the scrambled and the dictionary words to numbers and then used those to find the corrects words. Then by passing any online SHA256 converder you get the **FLAG**:
+
+{FLG:70c42b6d6818f3141865f10d1608f7a9763138554f6d5120662222189854231c}
```
I would suggest to enclose the flag into a "verbatim" section, such as

```text
{FLG:70c42b6d6818f3141865f10d1608f7a9763138554f6d5120662222189854231c}
```

so that the flag can be easily copied and then pasted into the CSC online platform

gmacario commented on training/week1/coding/scrambled-words/README.md:
```diff
@@ -0,0 +1,19 @@
+# Scrambled words
+
```
Please add the following metadata

* Category: Coding
* Points: 100

gteti committed:
Update README.md

Changed the README as requested. Added:
- Category 
- Points

gteti committed:
Update README.md
